src/components/ToDo.js

```diff
@@ -0,0 +1,53 @@
+import "./ToDo.css";
+import { FaRegTrashAlt } from "react-icons/fa";
+import { MdEdit } from "react-icons/md";
+
+export default function ToDo({
+  name,
+  onToggleStatus,
+  isDone,
+  onDeleteToDo,
+  handleNameChange,
+  index,
+}) {
+  function handleDeleteClick() {
+    onDeleteToDo(name);
+  }
+
+  function handleToggleClick() {
+    onToggleStatus(name);
+  }
+
+  const classForStatus = isDone ? "toDo--done" : "toDo--pending";
+  const classForText = isDone ? "text-moved" : "text-unmoved";
+
+  function handleNameClick() {
+    let isDone = false;
```

Comment:
This might be confusing with the `isDone` of the todo. Let's find a better name, like `changingNameIsDone`